tests/test_endpoint.py

```diff
@@ -18,16 +18,13 @@ def _close_callback():
     async def server_node(ep):
         if server_close_callback is True:
             ep.set_close_callback(_close_callback)
-        msg = bytearray(10)
-        await ep.recv(msg)
         if server_close_callback is False:
             await ep.close()
 
     async def client_node(port):
         ep = await ucp.create_endpoint(ucp.get_address(), port,)
         if server_close_callback is False:
             ep.set_close_callback(_close_callback)
-        await ep.send(bytearray(b"0" * 10))
         if server_close_callback is True:
             await ep.close()
 
```

Comment:
It's just not necessary for this test, so I decided to simplify it. I realize it would be best as a separate PR not to confuse, but I ended up doing it here to avoid forgetting later as I started adding the probe test as part of this before deciding to move it as a new file.

Reply:
Alright, make sense